mavedb_mapping/__init__.py

```diff
@@ -0,0 +1,19 @@
+
+import os
+from biocommons.seqrepo import SeqRepo
+
+from cool_seq_tool.data_sources.uta_database import UTADatabase
+
+from ga4gh.vrs.dataproxy import SeqRepoDataProxy
+
+from gene.query import QueryHandler
+from gene.database import create_db
+
+path_to_seqrepo = os.getenv("PATH_TO_SEQREPO", "/usr/local/share/seqrepo/latest")
+path_to_hg38_file = os.getenv("HG38_FILE", "hg38.2bit")  # default- in current directory
+data_file_path = "tests/data/"
+
+# utadb = UTADatabase(db_pwd="uta")
+qh = QueryHandler(create_db())
+sr = SeqRepo(path_to_seqrepo, writeable=True)
+dp = SeqRepoDataProxy(sr=sr)
```

Comment:
I would recommend putting resource initialization objects like this into a separate module -- if they're in `__init__.py`, then they'll be initialized every time someone calls anything from this package. That's probably their intention 99% of the time, but it's best practice to keep this kind of code out of the __init__ file to account for the 1% of cases where they might not need it.